src/invidious/videos.cr

```diff
@@ -355,11 +370,12 @@ struct Video
   getset_i64 likes
   getset_i64 views
 
+  # TODO: Make predicate_bool the default as to adhere to Crystal conventions
   getset_bool allowRatings
   getset_bool authorVerified
   getset_bool isFamilyFriendly
   getset_bool isListed
-  getset_bool isUpcoming
+  predicate_bool upcoming, isUpcoming
```

Comment:
Why are you only changing that one? There are multiple other `is*` methods!

Reply:
This PR is mostly to just fix the Ameba issues which for some reason does not detect the other methods as violating `Naming/PredicateName`

Hence the TODO comment a few lines above

https://github.com/iv-org/invidious/blob/b75e5cfba6b4683307a5755506deefaebee5c3dd/src/invidious/videos.cr#L373

Do you want me to also resolve those in this PR?

Reply:
No, please keep that for a future PR, I'm worried it could cause too much conflicts with other open PRs!